src/algebra/lie/basic.lean

```diff
@@ -560,7 +562,7 @@ instance lie_subalgebra_coe_submodule : has_coe (lie_subalgebra R L) (submodule
 
 /-- A Lie subalgebra forms a new Lie ring. -/
 instance lie_subalgebra_lie_ring (L' : lie_subalgebra R L) : lie_ring L' :=
-{ bracket      := λ x y, ⟨⁅x.val, y.val⁆, L'.lie_mem x.property y.property⟩,
+{ bracket      := λ x y, ⟨⁅x.val, y.val⁆, L'.lie_mem' x.property y.property⟩,
```

Comment:
This line probably works without being changed.

Reply:
It would except that the `lie_mem` lemma is defined below so I think I'll just use `lie_mem'` here.